config/filter.d/domino-smtp.conf

```diff
@@ -0,0 +1,41 @@
+# Fail2Ban configuration file for IBM Domino SMTP Server TASK to detect failed login attempts
+#
+# Author: Christian Brandlehner
+#
+# $Revision: 003 $
+#
+# Configuration:
+# Set the following Domino Server parameters in notes.ini:
+#       console_log_enabled=1
+#       log_sessions=2
+#
+# Depending on your locale you might have to tweak the date and time format so fail2ban can read the log
+
+#[INCLUDES]
+# Read common prefixes. If any customizations available -- read them from
+# common.local
+#before = common.conf
+
+[Definition]
+# Option:  failregex
+# Notes.:  regex to match the password failure messages in the logfile. The
+#          host must be matched by a group named "host". The tag "<HOST>" can
+#          be used for standard IP/hostname matching and is only an alias for
+#          (?:::f{4,6}:)?(?P<host>\S+)
+# Values:  TEXT
+#
+# Sample log entries (used different time formats and an extra sample with process info in front of date)
+# 01/23/2009 19:54:51   SMTP Server: Authentication failed for user bd ; connecting host 125.120.107.140
+# [28325:00010-3735542592] 22.06.2014 09:56:12   smtp: brian [74.208.72.28] authentication failure using internet password
+# 08.09.2014 06:14:27   smtp: drweb [81.82.251.231] authentication failure using internet password
+# 08.09.2014 06:14:27   SMTP Server: Authentication failed for user drweb ; connecting host 81.82.251.231
+
+failregex = .* SMTP Server: Authentication failed for user .* connecting host <HOST>
```

Comment:
please fix this bad expression (anchors left/right, don't use greedy catch-all):
```regex
__prefix = (?:\[[^\]]+\])?\s+
failregex = ^%(__prefix)sSMTP Server: Authentication failed for user .*? ; connecting host <HOST>$
```